### PR TITLE
Added options to change the Power and Mode Values

### PR DESCRIPTION
**Pull request by ThomasHFWright**
Hi there,

I'm trying to integrate with home assistant, and the default thermostat in that does not support the Power and Mode values you've got in your build. I added these as variables which can be overridden.
E.g. home assistant uses "OFF" and "ON" for Power, and there is not a very simple way of mapping these inbound and outbound to "Off" and "On". Easier just to use "OFF" and "ON" in MHI-AC-Ctrl

I also added a room name to the MQTT_PREFIX to make it more obvious what the syntax should be, so you can keep your HOSTNAME the same for each unit to group all the units together.
Lastly I added a separate OTA_NAME so that if you have multiple units in different rooms you can differentiate if the HOSTNAME value is the same for each.

### CODE REVIEW

File: src/MHI-AC-Ctrl.h [hunk 1]
```diff
@@ -23,6 +23,7 @@
 #define MODE_COOL = "Cool";
 #define MODE_FAN = "Fan";
 #define MODE_HEAT = "Heat";
+#define MODE_ERROR = "Stop";
```
Comment (b0rder) [inline src/MHI-AC-Ctrl.h]: Is it better to rename it to MODE_STOP?

Comment (ThomasHFWright) [reply src/MHI-AC-Ctrl.h]: I wasn't really sure what the "Stop" mode was used for, there's a comment in the ino file on line 106 that it is "for error operating data". So I named it like that.
Might be better if this was the following, but I didn't want to break any existing implementations.
`MODE_ERROR = "Error"`

**Comment by absalom-muc** [comment]
Hello ThomasHFWright,
Thank you very much for your contribution!
You address with your pull request different topics. I would like to give you feedback on the individual points: 
I like the idea simplifying the adaption for a specific use case (e.g. by using #define for MQTT messages), but I want to avoid changes of the main program due to these adaptions. E.g. the next framework might require different MQTT topic names etc.
Therefore, my favorite is to take up the idea from b0rder and to separate the core functionality (SPI communication) from the input / output (MQTT, OTA). That should allow a simpler adaption for different use cases or frameworks (e.g. Tasmota, Homie etc.).

With the introduction of “OTA_Name” and the changed “MQTT_PREFIX“ you enable the support of multiple AC devices. Not sure if more users have one AC device or multiple AC devices. Therefore, I would leave “MQTT_PREFIX” as it is, maybe use your proposal as a comment, e.g.
`#define MQTT_PREFIX HOSTNAME "/" // replace by e.g. „/Living-Room/“ when you have multiple ACs`
For OTA_NAME I would still use the HOSTNAME, e.g.
`#define OTA_NAME HOSTNAME`
Then it can be easy replaced for multiple devices, but the default is still the hostname.

**Comment by absalom-muc** [comment]
Thanks again for your input. Most of your topics are addressed in v2.0, therefore I close it here.

**Pull request closed without merging**